configure.ac

```diff
@@ -3615,71 +3615,31 @@ AC_SUBST(HAVE_LIBDISPATCH_RUNLOOP)
 # Check for libcurl
 # See DEPENDENCIES POLICY at the start of this file.
 #--------------------------------------------------------------------
-CURL_CONFIG="curl-config"
-AC_ARG_WITH(curl, AS_HELP_STRING([--with-curl=<DIR>],
-  [use curl installed in directory <DIR>]))
-if test "$with_curl" != ""; then
-  CURL_CONFIG="$with_curl/bin/curl-config"
-fi
 
 HAVE_LIBCURL=0
 curl_all=no
 AC_MSG_CHECKING([for libcurl])
-if eval $CURL_CONFIG --version 2>/dev/null >/dev/null; then
-  curl_ver=`$CURL_CONFIG --version | sed -e "s/libcurl //g"`
-  curl_maj=`echo $curl_ver | sed -e "s/^\(.*\)\.\(.*\)\.\(.*\)$/\1/"`
-  curl_min=`echo $curl_ver | sed -e "s/^\(.*\)\.\(.*\)\.\(.*\)$/\2/"`
-  if test $curl_maj -lt 7 -o \( $curl_maj -eq 7 -a $curl_min -lt 49 \); then
-    AC_MSG_RESULT([FAILED (version too old to use])
-  else
-    AC_MSG_RESULT(yes ... version $curl_ver)
-    AC_CHECK_HEADERS(curl/curl.h, curl_ok=yes, curl_ok=no)
+
+if test -n "$PKG_CONFIG"; then
+  if pkg-config --exists libcurl; then
```

Comment:
Shouldn't you be using $PKG_CONFIG here as well (and in line 3631)?

Reply:
You are right